extensions/big-data-cluster/README.md

```diff
@@ -1,17 +0,0 @@
-# Microsoft SQL Server big data cluster Extension for Azure Data Studio
-
-Welcome to Microsoft SQL Server big data cluster Extension for Azure Data Studio!
-
-## Code of Conduct
-
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
-
-## Privacy Statement
-
-The [Microsoft Enterprise and Developer Privacy Statement](https://privacy.microsoft.com/en-us/privacystatement) describes the privacy statement of this software.
-
-## License
-
-Copyright (c) Microsoft Corporation. All rights reserved.
-
-Licensed under the [Source EULA](https://raw.githubusercontent.com/Microsoft/azuredatastudio/master/LICENSE.txt).
```

Comment:
update the content instead of deleting it

Reply:
Content looks all good. Nothing to modify.